src/theme/Footer/Copyright/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
The `src/theme/Footer` folder was obtained via swizzling: https://docusaurus.io/docs/swizzling

Basically "ejected" footer component and customised it to our needs. Things like `dangerouslySetInnerHTML` come from original docusaurus code.